packages/core/src/html/NodeProcessor.ts

```diff
@@ -83,23 +99,24 @@ class NodeProcessor {
     }
   }
 
-  static _isText(node) {
+  static _isText(node: DomElement) {
     return node.type === 'text' || node.type === 'comment';
   }
 
   /*
    * FrontMatter collection
    */
-  _processFrontMatter(node, context) {
+  _processFrontMatter(node: DomElement, context: Context) {
     let currentFrontMatter = {};
     const frontMatter = cheerio(node);
     if (!context.processingOptions.omitFrontmatter && frontMatter.text().trim()) {
       // Retrieves the front matter from either the first frontmatter element
       // or from a frontmatter element that includes from another file
       // The latter case will result in the data being wrapped in a div
-      const frontMatterData = frontMatter.find('div').length
-        ? frontMatter.find('div')[0].children[0].data
-        : frontMatter[0].children[0].data;
+      const frontMatterIncludeDiv = frontMatter.find('div');
+      const frontMatterData = frontMatterIncludeDiv.length
+        ? ((frontMatterIncludeDiv[0] as DomElement).children as DomElement[])[0].data
+        : ((frontMatter[0] as DomElement).children as DomElement[])[0].data;
```

Comment:
I feel like there should be better ways to configure the typing here, let me know if any ideas 🙂 

Or we can defer to another PR to investigate, since functionally it is the same.

Reply:
I guess we can defer this to another PR to investigate. Can't think of a good idea right now...